context/app/static/js/components/detailPage/BulkDataTransfer/NoAccess.jsx

```diff
@@ -1,15 +1,30 @@
 import React from 'react';
+import InfoIcon from '@mui/icons-material/Info';
 
 import { DetailSectionPaper } from 'js/shared-styles/surfaces';
-import { StyledWarningIcon, WarningIconContainer, NoAccessContainer } from './style';
+import { SecondaryBackgroundTooltip } from 'js/shared-styles/tooltips';
+import { Header, StyledWarningIcon, IconContainer, NoAccessContainer, StyledExclamationIcon } from './style';
 
-function NoAccess({ children }) {
+const statusIcons = {
+  warning: <StyledWarningIcon />,
+  exclamation: <StyledExclamationIcon color="error" fontSize="small" />,
+};
+
+function NoAccess({ panelToUse }) {
+  const { status, children, title, tooltip } = panelToUse;
```

Comment:
Alternatively, the props could be `{ status, children, title, tooltip }`. This would let you call the component in BulkDataTransferPanels with `<NoAccess {...panelsToUse.error} />` and skip the destructuring step here, and would be in line with the `BulkDataTransferPanel` props.

Reply:
Thanks, done :)